resource_booking/templates/portal.xml

```diff
@@ -257,7 +257,7 @@
         </xpath>
     </template>
     <template
-        id="portal_my_home"
+        id="portal_my_home_resource_booking"
```

Comment:
No need of changing this id. In fact, the usual rule is to keep the parent id (but within this module).